lib/daterangepicker/daterangepicker.js

```diff
@@ -330,6 +337,9 @@
             singleDate: false
         });
 
+        this.timezone = options.timezone || DEFAULT_TIMEZONE;
```

Comment:
NABD, but as a part of [this test case](https://github.com/BrandwatchLtd/daterangepicker/pull/26/files#diff-c37d4ad1e617c697fc1eca93e9d34360R543) you can also check it the <code>daterangepicker</code> uses the default timezone (<code>UTC</code>).